cmd/mdatagen/templates/readme.md.tmpl

```diff
@@ -1,17 +1,20 @@
 <!-- status autogenerated section -->
-{{- if ne .Status.Stability "" }}
+{{- if len .Status.Stability }}
 | Status                   |           |
 | ------------------------ |-----------|
-| Stability                | [{{ .Status.Stability }}]   |
-{{- if ne .Status.Class "extension" }}
-| Supported pipeline types | {{ stringsJoin .Status.Pipelines ", " }}   |
+{{- $class := .Status.Class }}
+{{- $idx := 0 }}
+{{- range $stability, $value := .Status.Stability }}
+| {{ if not $idx }}Stability{{- end }}                | [{{ $stability }}] {{ if ne $class "extension" }}[{{ stringsJoin $value ", " }}] {{ end }}  |
```

Comment:
to keep the first column aligned and add `:`
```suggestion
| {{ if not $idx }}Stability{{ else }}         {{ end }}     | [{{ $stability }}]{{ if ne $class "extension" }}: [{{ stringsJoin $value ", " }}] {{ end }} |
```

Reply:
I ended up choosing not to put `:` as a separator, since i felt the `[]` were already providing this. Alternatively we could remove `[]` and add `:`? I don't have a strong preference here:

## 1
| Status | |
| -- | --|
| Stability | [beta](https://github.com/open-telemetry/opentelemetry-collector#beta): [metrics, logs] |
|  | [stable](https://github.com/open-telemetry/opentelemetry-collector#stable): [traces] |

## 2
| Status | |
| -- | --|
| Stability | [beta](https://github.com/open-telemetry/opentelemetry-collector#beta): metrics, logs |
|  | [stable](https://github.com/open-telemetry/opentelemetry-collector#stable): traces |

## 3
| Status | |
| -- | --|
| Stability | [beta](https://github.com/open-telemetry/opentelemetry-collector#beta) [metrics, logs] |
|  | [stable](https://github.com/open-telemetry/opentelemetry-collector#stable) [traces] |

I'm happy with either option 2 or 3. WDYT?

Reply:
I went w/ option 2.